README.md

```diff
@@ -21,11 +21,8 @@ materials science, quantum chemistry, and biology.
   - [Docker](#docker)
   - [From source](#from-source)
 - [Getting Started](#getting-started)
-- [Contributing to DeepChem](/CONTRIBUTING.md)
-  - [Code Style Guidelines](/CONTRIBUTING.md#code-style-guidelines)
-  - [Documentation Style Guidelines](/CONTRIBUTING.md#documentation-style-guidelines)
-  - [Gitter](#gitter)

```

Comment:
Could we add a link to the Gitter back in? It looks like it was incorrectly indented into the contribution section

Reply:
I add a link to the Gitter back in appropriate section.